Formula/newrelic-cli.rb

```diff
@@ -0,0 +1,19 @@
+class NewrelicCli < Formula
+  desc 'The New Relic Command-line Interface'
+  homepage 'https://github.com/newrelic/newrelic-cli'
+  url 'https://github.com/newrelic/newrelic-cli/archive/v0.2.3.tar.gz'
+  sha256 'f09d7e1705731578da79f0d9c204a6875edbd1d78cc6e69bd9f7d00dd9919986'
+  head 'https://github.com/newrelic/newrelic-cli.git'
+
+  depends_on 'go' => :build
+
+  def install
+    ENV['PROJECT_VER'] = version
+    system 'make', 'compile-only'
+    bin.install 'bin/darwin/newrelic'
+  end
+
+  test do
+    system "#{bin}/newrelic", 'config list'
```

Comment:
Pleas check some output here using `assert_match` and `shell_output`

Reply:
I've done as you ask to include a version check of the output.